cmake/GenerateBuiltinsList.cmake

```
@@ -1,3 +1,7 @@
+if (is-riscv)
+	set(bblg-prefix ${RISCV_EMULATOR} -P --) # Proxy through the emulator
```

```suggestion
  set(bblg-prefix ${RISCV_EMULATOR} -P --) # Proxy through the emulator
```
And please name it less abbrev.